internal/featuresupport/features_olm_operators.go

```diff
@@ -582,7 +583,12 @@ func (f *OpenShiftAIFeature) getIncompatibleArchitectures(_ *string) *[]models.A
 }
 
 func (f *OpenShiftAIFeature) getIncompatibleFeatures(string) *[]models.FeatureSupportLevelID {
-	return &[]models.FeatureSupportLevelID{}
+	return &[]models.FeatureSupportLevelID{
+		// These aren't directly incompatible with OpenShift AI, rather with ODF, but the feature support
+		// mechanism doesn't currently understand operator dependencies, so we need to add these explicitly.
```

Comment:
Should we raise a ticket to investigate how to factor in operator dependencies for the purpose of determining feature support?

It seems like this is an area that should be explored further, is there already an issue for this?

Reply:
Yes @paul-maidment , we have https://issues.redhat.com/browse/MGMT-19471 to address that.